.pre-commit-config.yaml

```
@@ -0,0 +1,8 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.14.2
+    hooks:
+      - id: helm-docs
+        args:
+          # Make the tool search for charts only under the `charts` directory
+          - --chart-search-root=charts
```

```suggestion
      - id: helm-docs
        args:
          - --chart-search-root=charts
```

Straight forward as is.